lib/components/ReferenceLine/constants.ts

```diff
@@ -4,3 +4,13 @@ export const defaultReferenceLineProps: ReferenceLineProps = {
   stroke: "inherit",
   className: "stroke-gray-300",
 };
+
+export const scatterReferenceLineXProps: ReferenceLineProps = {
```

Comment:
We could've created separate `ReferenceLine`, etc. components for `ScatterChart` but I think this makes more sense. 

when recharts v3 is around we're gonna do something like this  
```
 -- <ReferenceLine {...scatterReferenceLineXProps} />
++ <ReferenceLine variant="scatter" axis="x" />
```

Reply:
I don't think we use the reference line for Scatterplots btw

Reply:
Taking it back, I think this makes sense!